controller/workitem.go

```diff
@@ -214,9 +214,6 @@ func (c *WorkitemController) Delete(ctx *app.DeleteWorkitemContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, errs.Wrapf(err, "error deleting work item %s", ctx.ID))
 		}
-		if err := appl.WorkItemLinks().DeleteRelatedLinks(ctx, ctx.ID); err != nil {
```

Comment:
to clarify: the call `appl.WorkItemLinks().DeleteRelatedLinks(ctx, ctx.ID);` should not have been removed and it was restored in https://github.com/almighty/almighty-core/pull/956/commits/e7bff179b2ac5ff8ab66b921473e84c70e15a1fe